### ISSUE TITLE
make sure multi county application cannot fail

### ISSUE THREAD

**Issue by bengolder** [codeforamerica/intake#121]
``` python
Internal Server Error: /application/

AttributeError at /application/
'NoneType' object has no attribute 'build_form_class'

Request Method: GET
Request URL: https://clearmyrecord.codeforamerica.org/application/
Django Version: 1.9.7
Python Executable: /app/.heroku/python/bin/python
Python Version: 3.5.0
Python Path: ['/app', '/app/.heroku/python/bin', '/app/.heroku/python/lib/python3.5/site-packages/setuptools-23.1.0-py3.5.egg', '/app/.heroku/python/lib/python3.5/site-packages/pip-8.1.2-py3.5.egg', '/app', '/app/.heroku/python/lib/python35.zip', '/app/.heroku/python/lib/python3.5', '/app/.heroku/python/lib/python3.5/plat-linux', '/app/.heroku/python/lib/python3.5/lib-dynload', '/app/.heroku/python/lib/python3.5/site-packages']
Server time: Fri, 9 Sep 2016 03:10:55 +0000
Installed Applications:
['django.contrib.sites',
 'django.contrib.admin',
 'django.contrib.auth',
 'django.contrib.contenttypes',
 'django.contrib.sessions',
 'django.contrib.messages',
 'django.contrib.staticfiles',
 'intake',
 'user_accounts',
 'allauth',
 'allauth.account',
 'allauth.socialaccount',
 'django_jinja',
 'invitations',
 'storages',
 'formation',
 'debug_toolbar']
Installed Middleware:
['django.middleware.security.SecurityMiddleware',
 'django.contrib.sessions.middleware.SessionMiddleware',
 'django.middleware.common.CommonMiddleware',
 'django.middleware.csrf.CsrfViewMiddleware',
 'django.contrib.auth.middleware.AuthenticationMiddleware',
 'django.contrib.auth.middleware.SessionAuthenticationMiddleware',
 'django.contrib.messages.middleware.MessageMiddleware',
 'django.middleware.clickjacking.XFrameOptionsMiddleware']


Traceback:

File "/app/.heroku/python/lib/python3.5/site-packages/django/core/handlers/base.py" in get_response
  149.                     response = self.process_exception_by_middleware(e, request)

File "/app/.heroku/python/lib/python3.5/site-packages/django/core/handlers/base.py" in get_response
  147.                     response = wrapped_callback(request, *callback_args, **callback_kwargs)

File "/app/.heroku/python/lib/python3.5/site-packages/django/views/generic/base.py" in view
  68.             return self.dispatch(request, *args, **kwargs)

File "/app/.heroku/python/lib/python3.5/site-packages/django/views/generic/base.py" in dispatch
  88.         return handler(request, *args, **kwargs)

File "/app/.heroku/python/lib/python3.5/site-packages/django/views/generic/edit.py" in get
  213.         return self.render_to_response(self.get_context_data())

File "/app/intake/views.py" in get_context_data
  113.         context = super().get_context_data(*args, **kwargs)

File "/app/.heroku/python/lib/python3.5/site-packages/django/views/generic/edit.py" in get_context_data
  122.             kwargs['form'] = self.get_form()

File "/app/.heroku/python/lib/python3.5/site-packages/django/views/generic/edit.py" in get_form
  73.             form_class = self.get_form_class()

File "/app/intake/views.py" in get_form_class
  142.         return county_form_selector.get_combined_form_class(counties=counties)

File "/app/formation/combinable_base.py" in get_combined_form_class
  95.         return combined_spec.build_form_class(criteria, self.form_parent_class)

Exception Type: AttributeError at /application/
Exception Value: 'NoneType' object has no attribute 'build_form_class'
Request information:
GET: No GET data

POST: No POST data

FILES: No FILES data

META:
HTTP_TOTAL_ROUTE_TIME = '0'
HTTP_CONNECT_TIME = '1'
PATH_INFO = '/application/'
HTTP_CACHE_CONTROL = 'max-age=0'
wsgi.version =
SERVER_PROTOCOL = 'HTTP/1.1'
HTTP_REFERER = 'https://clearmyrecord.codeforamerica.org/apply/'
HTTP_X_FORWARDED_FOR = '172.0.106.119'
wsgi.multiprocess = True
wsgi.multithread = False
HTTP_UPGRADE_INSECURE_REQUESTS = '1'
SERVER_NAME = '0.0.0.0'
HTTP_COOKIE = '_gat=1; intercom-visitor-semaphore-bey83rgj=1; sessionid=l2ybe6nynv6caxrhqqe1x9qzn4kj5no8'
QUERY_STRING = ''
RAW_URI = '/application/'
wsgi.errors = <gunicorn.http.wsgi.WSGIErrorsWrapper object at 0x7f2be11d56a0>
REQUEST_METHOD = 'GET'
REMOTE_PORT = '48158'
HTTP_ACCEPT_ENCODING = 'gzip, deflate, sdch, br'
wsgi.file_wrapper = ''
HTTP_X_FORWARDED_PORT = '443'
HTTP_CONNECTION = 'close'
HTTP_USER_AGENT = 'Mozilla/5.0 (X11; CrOS x86_64 8350.68.0) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/52.0.2743.116 Safari/537.36'
HTTP_ACCEPT_LANGUAGE = 'en-GB,en-US;q=0.8,en;q=0.6'
wsgi.url_scheme = 'https'
HTTP_X_REQUEST_START = '1473390655219'
SERVER_SOFTWARE = 'gunicorn/19.6.0'
HTTP_HOST = 'clearmyrecord.codeforamerica.org'
HTTP_X_FORWARDED_PROTO = 'https'
HTTP_VIA = '1.1 vegur'
HTTP_ACCEPT = 'text/html,application/xhtml+xml,application/xml;q=0.9,image/webp,*/*;q=0.8'
wsgi.run_once = False
wsgi.input = <gunicorn.http.body.Body object at 0x7f2be11d5ef0>
SCRIPT_NAME = ''
```

**Comment by bengolder** [comment]
This is trivial to reproduce. Just submit hit "Apply" without selecting counties.

**Comment by bengolder** [comment]
closed with da6ea596bf53e8d6ab59fdf48b9d254415f91f91
